screens/CreateGame.js

```diff
@@ -72,20 +106,32 @@ function CreateGame({ navigation }) {
   }
 
   //saves a name game or returns an error
-  async function storeGame(gameInfo) {
+  async function storeGame() {
+
```

Comment:
Keep the time validation inside of create(). We should have validation logic BEFORE calling any async functions. This function should just focus on storing a game, not any input validation.